src/services/services.ts

```diff
@@ -1244,6 +1244,7 @@ namespace ts {
     }
 
     export interface EditorOptions {
+        BaseIndentSize?: number;

```

Comment:
Why does this structure have a different shape (optionality) to the EditorOptions above in protocol.d.ts? (Not your code, but the existing fields).